repository/CargoPackageManager-Minimal.package/CGOCargoProjectDependency.class/class/named.version..st

```diff
@@ -0,0 +1,7 @@
+instance-creation
+named: aName version: aVersion 
```

Comment:
a constraint is not built from a version but from a constraint string (e.g. '=1.2'). If you use a version, an error will be thrown